nomination/templates/nomination/project_listing.html

```diff
@@ -1,44 +1,44 @@
-{% extends "nomination/index.html" %}
+{% extends "nomination/base.html" %}
 
 {% block title %}Nomination Tool{% endblock %}
 
-{% block body_classes %}about-project{% endblock %}
-
-{% block project_nav %}{% endblock %}
-
-{% block main_nav %}
-            <div id="hd"><h1>Nomination Tool</h1>
-            <p id="other-projects"><a href="{% url 'nomination_help' %}">Help</a></p>
-            <ul id="navigation">
-                <li><a href="{% url 'project_listing' %}">Home</a></li>
-                <li><a href="{% url 'nomination_about' %}">About this Tool</a></li>
-            </ul>
-            </div>
-        {% endblock %}
-
 {% block content %}
 
-        <div class="field-wrapper">
-        <h2>Active Projects</h2>&nbsp;
-        <p>
-        {% for project in active_list %}
-            <a href="{% url 'project_urls' project.project_slug %}">
-                {{ project.project_name }}
-            </a><br/>
-        {% endfor %}
-        </p>
-        <h2>Past Projects</h2>&nbsp;
-        <p>
-        {% for project in past_list %}
-            <a href="{% url 'project_urls' project.project_slug %}">
-                {{ project.project_name }}
-            </a><br/>
-        {% endfor %}
-        </p>
+    <div class="panel-group">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Active Projects</h4>
+            </div>
+            <div class="panel-body">
+                {% for project in active_list %}
+                    <a href="{% url 'project_urls' project.project_slug %}">
+                        {{ project.project_name }}
+                    </a><br>
```

Comment:
I would use an unordered list for the projects rather than iterating through them with `<br>` tags to put them on separate lines. A list would be more semantic for this purpose.